lua/ulx/modules/sh/cfc_speed.lua

```diff
@@ -0,0 +1,22 @@
+CFCUlxCommands.speed = CFCUlxCommands.speed or {}
+local cmd = CFCUlxCommands.speed
+
+CATEGORY_NAME = "Fun"
+                    
+function cmd.speedPlayers( callingPlayer, targetPlayers, amount )
+    for _, ply in pairs( targetPlayers ) do
+        local inPvp = hook.Run( "CFC_Ulxcommands_SpeedPvpCheck", ply )
```

Comment:
if the point of using hooks like this is to decouple the ulx command addon from the pvp addon a more generic hook name would be better e.g. `CFC_ULXCommands_CanApplySpeedModifier`